### PR TITLE
Improve docs for std.range.take

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: std/range/package.d [hunk 1]
```diff
@@ -1788,8 +1788,17 @@ if (isRandomAccessRange!(Unqual!R) && hasLength!(Unqual!R))
 
 /**
 Lazily takes only up to $(D n) elements of a range. This is
-particularly useful when using with infinite ranges. If the range
-offers random access and $(D length), $(D Take) offers them as well.
+particularly useful when using with infinite ranges. Can also
+be used to provide length information to an input range.
+
+Params:
+    r = an input range
```
Comment (burner) [inline std/range/package.d]: same as https://github.com/dlang/phobos/pull/4444

**Comment by JackStouffer** [comment]
@burner Fixed

**Comment by burner** [comment]
LGTM

**Comment by burner** [comment]
Auto-merge toggled on

**Pull request merged**